app/js/services/individuService.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular.module('ddsCommon').service('IndividuService', function($filter, situationsPro, nationalites) {
-    var statutsSpecifiques = _.filter(situationsPro, 'isStatutSpecifique');
+    var statutsSpecifiques = situationsPro;
```

Comment:
Yep, `situationsPro` was containing both the "Particular situation" options asked for each individual in a simulation, and other informations captured just to fill the cerfa. As we don't need the latter anymore, we don't have to do the distinction.
What kind of refactor are you suggesting ? Renaming `situationsPro` to `particularSituation` everywhere ? Or just removing this useless intermediate variables ?